types/nightwatch-options.d.ts

```diff
@@ -129,6 +129,8 @@ export interface NightwatchScreenshotOptions {
 }
 
 export interface NightwatchOptions {
+
+	tag_filter?: string | string[]
```

Comment:
Can you move this to be below `skiptags` property? Also, can you add a TSDoc for this property? It can be something like: "Tag(s) used/to be used during test execution.".

Do the same in `@types/nightwatch` as well.

Reply:
ok @garg3133 i will.